src/Symfony/Bridge/PhpUnit/Legacy/ForwardCompatTestTraitForV7.php

```diff
@@ -18,8 +18,6 @@
  */
 trait ForwardCompatTestTraitForV7
 {
-    use ForwardCompatTestTraitForV5;
```

Comment:
I'm not sure this class is usefull anymore @nicolas-grekas .

`ForwardCompatTestTraitForV5` provides: 
- createMock (changed in PhpUnit 5)
- assertIsXXX (introduced in PhpUnit 7.5)
- expectException (introduced in PhpUnit 5)

Which make `ForwardCompatTestTraitForV5` the helper when running tests with phpunit version `4`, `5`, `6`.

Since `setUp` and `tearDown` methods are removed (not used anymore since simple_phpunit is able to remove typehint from php 8) the trait `ForwardCompatTestTraitForV7` don't have to extends `ForwardCompatTestTraitForV5` anymore and don't have to redefine the method `createMock` anymore. 

Am I right?

what's about
```
ForwardCompatTestTraitFor4 { // used when installed version of phpunit < 5 could be renamed into ForwardCompatTestTraitPolyfill5
  use ForwardCompatTestTraitForV6
  createMock;
  expectException;
}
ForwardCompatTestTraitForV6 { // used when installed version of phpunit < 7
  assertIsXXX;
}
```